src/components/MapView.js

```diff
@@ -103,6 +129,38 @@ class MapView extends Component {
                                         onClick={() => this.setState({popupInfo: report})}
                                 />
                             </Layer>))}
+                    <div>
+                        <Fab  aria-label="Navigation" className="navContainer" size="small">
+                            <NavigationIcon  onClick={() => this.setState({viewport: {
+                                    center: [-122.335167, 47.608013],
+                                    zoom: [10],
+                                }})}/>
+                        </Fab>
+                    </div>
+                    <br/>
+                    <div>
+                        <Fab  aria-label="Toggle" className="mapListToggle" size="small">
+                            <List onClick={() => history.push('/list')}/>
+                        </Fab>
+                    </div>
+                    <div>
+                        <Fab  aria-label="Legend" className="legendContainer" size="small">
+                            <Help onClick={() => this.setState({legend: true})}/>
+                        </Fab>
+                    </div>
+                    <div >
+                        <Dialog
```

Comment:
In the comps, this doesn't really look like a dialog. Specifically, it looks like you can interact with the map while the legend is open, and then click the "X" in the upper-right corner to close the legend.

Maybe we could consider using a `Card` or some other component instead here?